.github/workflows/publish.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.7, 3.8 ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Patch for Python 3.6
+      if: matrix.python-version == 3.6
+      run: |
+        pip install strip-hints==0.1.9
+        ./patch-python3.6.sh
+
+    - name: Install dependencies
+      run: |
+        pip install -r requirements.txt
+
+    - name: Install dev dependencies
+      run: |
+        pip install -r dev_requirements.txt
+
+    - name: Test
+      run: |
+        pytest
+
+    - name: Build package
+      run: |
+        PYTHON_VERSION=${{ matrix.python-version }}
+        python setup.py bdist_wheel --python-tag py${PYTHON_VERSION//.}
+
+    - name: Test install package
+      run: |
+        mkdir test_install
+        cd test_install
+
+        pip install virtualenv
+        virtualenv installable
+        source installable/bin/activate
+
+        pip install $(find ${GITHUB_WORKSPACE}/dist -iname "*.whl")
```

Comment:
Consider alternative `pip install $(ls dist)`

Reply:
Could also do `pip install $(ls dist/*.whl)`

Reply:
what a flip
tried again now
I used find because I thought ls only enumerated the files and didn't show the pathname relative to pwd but sedär I must have set up the experiment I did in a bad way
(if ls where to have worked in that way, there would have been the distribution path used twice)